### PR TITLE
Some fixes and information about push Capabilities

### PR DESCRIPTION
**Pull request by jcesarmobile**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,9 @@ and then:
 pod install
 ```
 
-to install your dependencies
+to install your dependencies.
+
+Finally, inside Xcode, go to the Capabilities section for your target and switch Push Notifications on.
```
Comment (corinnekrych) [inline README.md]: > Note: Since **iOS10**, inside Xcode, go to ...

@jcesarmobile  what about making clear it's ios10 requirement?

**Comment by jcesarmobile** [comment]
@corinnekrych @aidenkeating can you take a look?

File: README.md (hunk 1 above)
Comment (jcesarmobile) [inline README.md]: No, it's Xcode 8 requirement, and we have to use Xcode 8 as we are using Swift 3 now

**Comment by corinnekrych** [comment]
+1
Merging

**Pull request merged**